src/elements/content-explorer/ContentExplorer.js

```diff
@@ -240,6 +245,8 @@ class ContentExplorer extends Component<Props, State> {
             currentPageSize: initialPageSize,
             errorCode: '',
             focusedRow: 0,
+            gridColumnCount: DEFAULT_GRID_VIEW_COLUMNS,
+            maxGridColumnCountForWidth: GRID_VIEW_MAX_COLUMNS,
```

Comment:
There are two different maximums at play here.  `GRID_VIEW_MAX_COLUMNS` is the maximum number of columns we want to show in the GridView, regardless of the width.  `maxGridColumnCountForWidth` is the maximum number of columns that can be displayed based on the given width.  `maxGridColumnCountForWidth` gets updated based on the width of the `AutoSizer` containing `GridView`.

Reply:
minor but could we make the naming more consistent?